openhands/llm/async_llm.py

```diff
@@ -0,0 +1,107 @@
+import asyncio
+from functools import partial
+
+from litellm import completion as litellm_acompletion
```

Comment:
This is probaby a merge error when restructuring files and slipped through:
it should be `acompletion`, not `completion`.